### PR TITLE
Wsteam1 768 live label existing

### PR DESCRIPTION
**Pull request by HarveyPeachey**
Resolves JIRA [number]
Before:
image
After:
image

Overall changes
======
_A very high-level summary of easily-reproducible changes that can be understood by non-devs, and why these changes where made._

Code changes
======

- _A bullet point list of key code changes that have been made._

Testing
======
1. _List the steps used to test this PR._

Helpful Links
======
_Add Links to useful resources related to this PR if applicable._

[Coding Standards](https://github.com/bbc/simorgh/blob/latest/docs/Coding-Standards/README.md)

[Repository use guidelines](https://github.com/bbc/simorgh-infrastructure/blob/latest/documentation/repository-guidelines.md)

### CODE REVIEW

File: ws-nextjs-app/pages/[service]/live/[id]/Header/index.tsx [hunk 1]
```diff
@@ -36,24 +28,17 @@ const Header = ({
...
-                  {`${liveLabel}, `}
-                </VisuallyHiddenText>
-              </>
-            ) : null}
-            <span
-              css={[styles.title, showLiveLabel && styles.layoutWithLiveLabel]}
-            >
-              {title}
-            </span>
+              <LiveLabel>{title}</LiveLabel>
```
Comment (karinathomasbbc) [inline ws-nextjs-app/pages/[service]/live/[id]/Header/index.tsx:31]: What would it look like if we didn't put the title as a child of the component, but rather render it on its own.

```suggestion
              <LiveLabel />
```

Comment (karinathomasbbc) [reply ws-nextjs-app/pages/[service]/live/[id]/Header/index.tsx:31]: What if we also used the (clatty hack) first-promo class:

```suggestion
              <LiveLabel className='first-promo' >
```

Comment (HarveyPeachey) [reply ws-nextjs-app/pages/[service]/live/[id]/Header/index.tsx:31]: We need it nested as a child for screen reader/a11y purposes I think https://github.com/bbc/simorgh/blob/92dc6472330112d78af808bd77b30adca4aa371f/src/app/components/LiveLabel/index.tsx#L71-L77

Again it feels weird adding first-promo when this isn't a promo, we could rename the class, but again it adds complexity to the live label component for specific live page styling

Comment (karinathomasbbc) [reply ws-nextjs-app/pages/[service]/live/[id]/Header/index.tsx:31]: Yeah true, we could rename the class to be better.. but like you said, it's starting to add more and more complexity to the component. Just for completeness, can we add the first-promo class to see if it matches the live page designs? Or are we just wasting time going down this road trying to make it work?

Comment (HarveyPeachey) [reply ws-nextjs-app/pages/[service]/live/[id]/Header/index.tsx:31]: So the [design](https://www.figma.com/file/ozHsWG5R9tETw6lBOU740V/Live-mvp-header-(text-only)---handover?node-id=402%3A37066&mode=dev) has a width and height of 1.5rem, which is 24px for all breakpoints which is slightly different from first-promo

Comment (karinathomasbbc) [reply ws-nextjs-app/pages/[service]/live/[id]/Header/index.tsx:31]: Ah OK then so, thanks for explaining. Let's just bin this, and focus on the other PR. Thanks for trying this out, and apologies for making you do extra work. I just wanted to make sure that we had confirmed 100% that using the label as is was not going to be suitable before committing to the refactor.

**Pull request closed without merging**